jrnl/util.py

```diff
@@ -180,9 +198,20 @@ def get_text_from_editor(config, template=""):
     return raw
 
 
-def colorize(string):
-    """Returns the string wrapped in cyan ANSI escape"""
-    return u"\033[36m{}\033[39m".format(string)
+def colorize(string, color, bold=False):
+    """Returns the string colored with colorama.Fore.color. If the color set by
+    the user is "NONE", it returns the string without any modification. Otherwise,
+    If the color set by the user doesn't exist in the colorama.Fore attributes,
+    the colorization is done with WHITE."""
+    upper_color = color.upper()
+    if upper_color == "NONE":
+        return string
+
+    color_escape = getattr(colorama.Fore, upper_color, colorama.Fore.WHITE)
```

Comment:
I think the default should just be no color at all. This way we keep backwards compatibility and we don't confuse users who have set their terminal background color to white.

Reply:
White is the default for an invalid color, but I can change the color defaults to none when the config is created.

Reply:
Ah, I just realized somebody with an old config would just get the new default values as well.
Yea that would probably be better.

Reply:
Done.